### PR TITLE
Adb 813

### PR DESCRIPTION
**Pull request by hiteshdigit101**
ADB-813
ADB-809
ADB-846
ADB-843
ADB-906

### CODE REVIEW

File: src/components/adb-nad-account-register/index.tsx [hunk 1]
```diff
@@ -1,38 +1,92 @@
...
-        <View style={styles.paddingVertical}/>
-        <Button label={i18n.t('adb_payment.nad_Registration.viewAvailableId') ?? "View Available ID"} onPress={onViewId} />
-      </View>}
+          <Button
+            label={
+              i18n.t("adb_payment.nad_Registration.maybeLater") ?? "Maybe Later"
+            }
+            onPress={onMaybeLater}
+            background={"white"}
+            labelColor={"#1B1B1B"}
```
Comment (sharmal33) [inline src/components/adb-nad-account-register/index.tsx:70]: plz use themes colors

File: src/components/adb-nad-account-register/index.tsx [hunk 2]
```diff
@@ -41,59 +95,59 @@ const styles = StyleSheet.create({
...
   },
   paddingVertical: {
-    height: 10
+    height: 10,
   },
   noteSection: {
     marginTop: 20,
     marginBottom: 15,
-    backgroundColor:'#F5F5F5',
+    backgroundColor: "#F5F5F5",
```
Comment (sharmal33) [inline src/components/adb-nad-account-register/index.tsx:106]: plz use themes colors

File: src/components/adb-nad-account-register/index.tsx [hunk 3]
```diff
@@ -41,59 +95,59 @@ const styles = StyleSheet.create({
...
     borderRadius: 5,
     padding: 15,
   },
   imageContent: {
     borderRadius: 30,
-    backgroundColor: '#D9D9D9',
+    backgroundColor: "#D9D9D9",
```
Comment (sharmal33) [inline src/components/adb-nad-account-register/index.tsx:112]: plz use themes colors

File: src/components/adb-nad-account-register/index.tsx [hunk 4]
```diff
@@ -41,59 +95,59 @@ const styles = StyleSheet.create({
...
   },
   content: {
     flex: 1,
     marginTop: 25,
   },
   title: {
     fontSize: 24,
-    color: '#1B1B1B',
-    fontFamily: fonts.semiBold
+    color: "#1B1B1B",
```
Comment (sharmal33) [inline src/components/adb-nad-account-register/index.tsx:142]: plz use themes colors

File: src/components/adb-nad-account-register/index.tsx [hunk 5]
```diff
@@ -41,59 +95,59 @@ const styles = StyleSheet.create({
...
+    fontFamily: fonts.semiBold,
   },
   subTitle: {
     fontSize: 14,
-    color: '#1B1B1B',
```
Comment (sharmal33) [inline src/components/adb-nad-account-register/index.tsx:93]: plz use themes colors

File: src/components/proxy-transfer-component/error-modal.tsx [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+        style={{ height: 75, width: 75, alignSelf: "center" }}
+      />
+      <Text
+        style={{
+          color: "#000000",
+          fontSize: 24,
+          fontWeight: "700",
+          alignSelf: "center",
+          marginTop: 30,
+        }}
```
Comment (sharmal33) [inline src/components/proxy-transfer-component/error-modal.tsx:51]: don't write inline styles

File: src/components/proxy-transfer-component/error-modal.tsx [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+          : i18n.t("adb_payment.proxy_transfer.duitnow_failed")}
+      </Text>
+      <Text
+        style={{
+          fontSize: 15,
+          fontWeight: "300",
+          alignSelf: "center",
+          marginTop: 15,
+          textAlign: "center",
+        }}
```
Comment (sharmal33) [inline src/components/proxy-transfer-component/error-modal.tsx:64]: don't write inline styles

**Review by sharmal33 — CHANGES_REQUESTED** [review]
- don't use inline styles 
- plz use theme component colors

File: src/components/adb-nad-account-register/index.tsx (hunk 1 above)
Comment (hiteshdigit101) [reply src/components/adb-nad-account-register/index.tsx:70]: Updated to theme component

File: src/components/adb-nad-account-register/index.tsx (hunk 2 above)
Comment (hiteshdigit101) [reply src/components/adb-nad-account-register/index.tsx:106]: Updated to theme colors

File: src/components/adb-nad-account-register/index.tsx (hunk 3 above)
Comment (hiteshdigit101) [reply src/components/adb-nad-account-register/index.tsx:112]: Updated to theme component

File: src/components/adb-nad-account-register/index.tsx (hunk 4 above)
Comment (hiteshdigit101) [reply src/components/adb-nad-account-register/index.tsx:142]: Updated to theme colors

File: src/components/adb-nad-account-register/index.tsx (hunk 5 above)
Comment (hiteshdigit101) [reply src/components/adb-nad-account-register/index.tsx:93]: Updated to theme colors

File: src/components/proxy-transfer-component/error-modal.tsx (hunk 6 above)
Comment (hiteshdigit101) [reply src/components/proxy-transfer-component/error-modal.tsx:51]: Moved to style sheet

File: src/components/proxy-transfer-component/error-modal.tsx (hunk 7 above)
Comment (hiteshdigit101) [reply src/components/proxy-transfer-component/error-modal.tsx:64]: Moved to style sheet

**Review by sharmal33 — APPROVED** [review]

**Pull request merged**